app/src/main/res/layout/activity_login.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I did not see the preview, but many times, instead of wrap_content we use 0dp to leave the layout relative for all kinds of devices.

Reply:
![image](https://user-images.githubusercontent.com/18317062/73386222-46092000-42ad-11ea-8e6b-a2d98ac6379f.png)
I setted the first view related to parent view, and the others to the one on top of each one, is it wrong?

Reply:
have you tried differents size of screens?